### PR TITLE
Move CommandStaticTimeIT to Sandbox integration tests

### LINKED ISSUE #2848: Port CommandStaticTimeIT to the Ledger API Test Tool
Port `CommandStaticTimeIT` to the Ledger API Test Tool.

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
Closes #2848

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags, if appropriate
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala [hunk 1]
```diff
@@ -179,24 +246,39 @@ class CommandStaticTimeIT
...
+                  unwrappedLedgerId,
+                  Some(txEndOffset),
+                  None,
+                  Some(TransactionFilter(
+                    Map(submitRequest.commands.value.party -> Filters.defaultInstance)))),
+                TransactionServiceGrpc
+                  .stub(channel)
+                  .getTransactions
+              )
+              .flatMapConcat(res => Source.fromIterator(() => res.transactions.iterator))
```
Comment (SamirTalwar) [inline ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala]: Why not just `.flatMapConcat(Source(_.transactions))`?

File: ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala [hunk 2]
```diff
@@ -102,33 +161,38 @@ class CommandStaticTimeIT
...
 
   "Command and Time Services" when {
 
     "ledger effective time is within acceptance window" should {
 
-      "commands should be accepted" in allFixtures { ctx =>
+      "commands should be accepted" in {
         for {
-          commandClient <- ctx.commandClient()
+          commandClient <- commandClient
```
Comment (SamirTalwar) [inline ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala]: Please rename one of these. Preferably the function. Call it `newCommandClient` or something.

I also think it'd be nice if it had parentheses. It's not a pure function, exactly—it uses the materializer.

File: ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala [hunk 3]
```diff
@@ -64,33 +115,41 @@ class CommandStaticTimeIT
...
       .update(
         _.commands.ledgerEffectiveTime := let,
         _.commands.maximumRecordTime := mrt,
-        _.commands.ledgerId := ctx.ledgerId.unwrap,
+        _.commands.ledgerId := unwrappedLedgerId,
         _.commands.commandId := newCommandId()
       )
   }
 
   // format: off
```
Comment (SamirTalwar) [inline ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala]: Is there a reason we're not letting the formatter do its thing?

File: ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala (hunk 1 above)
Comment (stefanobaghino-da) [reply ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala]: `Source` wants `scala.collection.immutable.Iterable[A]` and the value returned by the gRPC call is `scala.collection.Seq[A]` which is `<: scala.collection.Iterable[A]` but cannot be upcast to a `scala.collection.immutable.Iterable[A]`. I was a bit confused as well, but I can have a second look. :slightly_smiling_face:

File: ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala (hunk 2 above)
Comment (stefanobaghino-da) [reply ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala]: Makes a lot of sense, thanks.

File: ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala (hunk 3 above)
Comment (stefanobaghino-da) [reply ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala]: I can disable this and see if it works, all I've done was adding a missing `// format: off`. :wink:

Comment (stefanobaghino-da) [reply ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala]: My best guess is a `scalafmt` search state explosion, but probably this has been mitigated by your recent dependency bump and it's worth trying. :slightly_smiling_face:

Comment (stefanobaghino-da) [reply ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala]: > Search state exploded around line 135
Confirmed, I'll have to leave this as is, but at least the rest of the file will be formatted automatically. :slightly_smiling_face:

File: ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala (hunk 1 above)
Comment (stefanobaghino-da) [reply ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala]: Addressed by https://github.com/digital-asset/daml/pull/3874/commits/ee05a8ccee3e4c282f52449d9536e91d23442ad0

File: ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala (hunk 2 above)
Comment (stefanobaghino-da) [reply ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala]: Addressed by https://github.com/digital-asset/daml/pull/3874/commits/2d27ff8655ae1e109a97201be518807c241f1cf5

File: ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala (hunk 3 above)
Comment (SamirTalwar) [reply ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala]: Fair. 😄

**Pull request merged**